frontend/public/components/storage/create-pvc.tsx

```diff
@@ -184,21 +152,23 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
         </label>
         <div className="form-group">
           {this.state.accessModeRadios.map(radio => {
-            const checked = radio.value === this.state.accessMode;
+            const disabled = storageClassAccessMode ? true : false;
```

Comment:
nit: this is fine of course, but we generally use the `!!` idiom to coerce to bool

```suggestion
            const disabled = !!storageClassAccessMode;
```